src/hotspot/share/logging/logAsyncWriter.hpp

```diff
@@ -135,7 +55,95 @@ typedef KVHashtable<LogFileStreamOutput*, uint32_t, mtLogging> AsyncLogMap;
 // change the logging configuration via jcmd, LogConfiguration::configure_output() calls flush() under the protection of the
 // ConfigurationLock. In addition flush() is called during JVM termination, via LogConfiguration::finalize.
 class AsyncLogWriter : public NonJavaThread {
+  friend class AsyncLogTest;
+  friend class AsyncLogTest_logBuffer_vm_Test;
   class AsyncLogLocker;
+  using AsyncLogMap = ResourceHashtable<LogFileStreamOutput*, uint32_t, primitive_hash<LogFileStreamOutput*>,
+                                        primitive_equals<LogFileStreamOutput*>, 17,
+                                        ResourceObj::C_HEAP, mtLogging>;
```

Comment:
These arguments should be on the new lines each, I think? This would match the upstream code better.

Reply:
hi, @shipilev 
Do we have linter rules for this?

Reply:
even in tip of jdk, I still can find code like this: 
https://github.com/openjdk/jdk/blob/master/src/hotspot/share/classfile/classLoaderStats.hpp#L114